lib/ts/ink_args.cc

```diff
@@ -320,7 +320,7 @@ usage(const ArgumentDescription *argument_descriptions, unsigned n_argument_desc
     case 'f':
     case 'F':
       if (argument_descriptions[i].location) {
-        fprintf(stderr, " %-9s", *(int *)argument_descriptions[i].location ? "true " : "false");
+        fprintf(stderr, " %-9s", *(bool *)argument_descriptions[i].location ? "true " : "false");
```

Comment:
Every other place that has a flag argument uses an ``int`` for the flag. This should be consistent, so if you change the flag type, you need to change it everywhere.